js/isir-frame-io.js

```diff
@@ -6,26 +6,43 @@
 
 // Given a collection of [Files]() text files containing one or more fixed-width ISIR frames
 // Read each line, using CR/LF line endings, and return all non-blank, non SAIG routing codes.
-export async function read_isir_frames_from_filelist(file_list_src) {
+export async function read_isir_frames(filelist_src) {
     let isir_samples = []
-    for await (let isir_frame of aiter_read_isir_frames_from_filelist(file_list_src))
+    for await (let isir_frame of aiter_read_isir_frames(filelist_src))
```

Comment:
Prefer `const` over `let` where possible. Do you have plans to add linting? `prefer-const` is configurable in ESLint and would catch this.